ui/pages/team/index.tsx

```diff
@@ -40,20 +40,18 @@ export default function TeamJoin() {
       />
       <Container>
         <ContentLayout
-          header="Join MoonDAO"
+          header="Join the Network"
           headerSize="max(20px, 3vw)"
           mainPadding
           mode="compact"
           popOverEffect={false}
           isProfile
           description={
             <>
-              Be part of the first open-source, interplanetary network state
-              dedicated to establishing a permanent human presence on the Moon
-              and beyond. Registration is currently invite-only, but you can
-              send an email to{' '}
-              <Link href="mailto:info@moondao.com">info@moondao.com</Link> if
-              you think you'd be a good fit.
+              The Space Acceleration Network is an onchain startup society 
+              focused on building a permanent settlement on the Moon and beyond. 
+              Please submit an application if you think you'd be a good fit for this 
+              exclusive network.
```

Comment:
Do we want to market it as "exclusive" -- it's currently "exclusive" but over time I think we'll want to give people credentials and stamps and allow anyone to create a profile. Let's not add exclusive for now IMO.